docs/source/status.rst

```diff
@@ -0,0 +1,21 @@
+.. _status:
+
+---------------------
+Implementation Status
+---------------------
+
+The GA4GH server is currently under active development, and not all features
```

Comment:
"not all features may be fully functional yet" reads a bit funny. Perhaps "several features are not yet fully functional"?